pkg/httpget/httpget.go

```diff
@@ -0,0 +1,36 @@
+package httpget
```

Comment:
Please add boilerplate license headers at the beginning.

Reply:
Done.